keyboards/kbdfans/kbd75round1/readme.md

```diff
@@ -0,0 +1,18 @@
+# KBD75 round 1 & 2
+
+QMK Firmware is not the official firmware for the KBD75 round 1 & 2. 
+
+* Keyboard Maintainer: Anh Nguyen
+* Hardware Supported: KBD75 round 1 & 2 PCB
+
```

Comment:
Please add a link to a pcb image, and some text describing how to discern this vs. other "kbd75" variants.